README.md

```diff
@@ -39,7 +39,13 @@ npm install opent2t-translator-com-wink-thermostat
 ```
 2. First step is to do the onboarding. Wink communicates via the hub so you need to set that up first
 ```bash
-node index.js -o opent2t-translator-com-wink-hub -h WinkHub
+node_modules\.bin\opent2t-cli -o opent2t-translator-com-wink-hub -h WinkHub
+```
+
+Or, if installed globally
```

Comment:
It looks for the translator packages in the node_modules of the current directory.

---
In reply to: [105346389](https://github.com/openT2T/opent2t-cli/pull/28#discussion_r105346389) [](ancestors = 105346389)